src/containers/map/component.tsx

```diff
@@ -44,10 +50,13 @@ export const DEFAULT_PROPS = {
 
 const MapContainer = ({ mapId }: { mapId: string }) => {
   const basemap = useRecoilValue(basemapAtom);
-  const locationBounds = useRecoilValue(locationBoundsAtom);
+  const [{ enabled: isDrawingToolEnabled, uploadedGeojson, customGeojson }, setDrawingToolState] =
+    useRecoilState(drawingToolAtom);
+  const [locationBounds, setLocationBounds] = useRecoilState(locationBoundsAtom);
   const [URLBounds, setURLBounds] = useRecoilState(URLboundsAtom);
 
   const [activeWidgets, setActiveWidgets] = useRecoilState(activeWidgetsAtom);
+  const [, setAnalysisState] = useRecoilState(analysisAtom);
```

Comment:
why don't you use useSetRecoilState?

Reply:
Feeling lazy to import `useSetRecoilState `, and `useRecoilState` was already there. They are the same thing.